imports/api/seed/jobs.js

```diff
@@ -0,0 +1,13 @@
+import faker from 'faker';
```

Comment:
I'm creating the files `imports/api/seed/jobs.js` and `imports/api/seed/users.js` to seed the database during the test, because on testing we will always work on a clean DB.

You might be wondering what the heck is this `faker.name.jobTitle()`, this is the library [faker.js](https://github.com/marak/Faker.js/), used to generate random data to seed the database. It is useful because if we always add the same data to the DB we could be missing some errors, so seeding everytime with new data allows us to test every test with a different user, job, or whatever the collection is